converter/__main__.py

```diff
@@ -1,47 +1,12 @@
 # -*- encoding: utf-8 -*-
-import sys
 
-from .arguments import parse_arguments
-from .converter import get_convertor_function
-from .exception import TemperatureError, ConverterError
-from .utils import is_float
-from .interactive import get_interactive_function, CELSIUM_INPUT, FAH_INPUT, QUIT_INPUT
+from converter.exception import TemperatureError, ConverterError
+from converter.factory import ConvertorContextFactory
 
-args = parse_arguments()
-
-
-if args.verbose:
-    print(str(sys.argv) + " It's verbose argument")
-
-
-def get_temperature(args) -> float:
-    if args.temperature is None:
-        raise ConverterError("Provide value for converting")
-    return args.temperature
-
-
-if args.fahrenheit or args.celsius:
-    converter_function = get_convertor_function(args.fahrenheit, args.celsius)
-    try:
-        temperature = get_temperature(args)
-        print(converter_function(temperature))
-    except (TemperatureError, ConverterError) as error:
-        print(f"{error}")
-    quit()
-
-
-answer = str(input("What temperature you want convert?: ")).lower()
-while answer not in (CELSIUM_INPUT + FAH_INPUT + QUIT_INPUT):
-    answer = str(input("What temperature you want convert?: ")).lower()
-
-
-if answer in (CELSIUM_INPUT + FAH_INPUT):
-    conversion_function, temperature = get_interactive_function(answer)
-    if is_float(temperature):
-        try:
-            print(conversion_function(float(temperature)))
-        except TemperatureError as error:
-            print(f"{error}")
-        quit()
-    else:
-        print(f"ERROR: {temperature} is not convertible to float type")
+converter_context = ConvertorContextFactory().create()
+converter_function = converter_context.get_converter_function()
+try:
+    temperature = converter_context.get_temperature()
+    print(converter_function(temperature))
```

Comment:
@MariaMaximova, now, you don't need separate veriable `converter_function` here, you can directly call `get_converter_function`;
More over, I suggest renaming method to just `get_converter`.
```suggestion
    print(converter_context.get_converter()(temperature))
```